content/en/docs/quick-start/minimal-kubesphere-on-k8s.md

```diff
@@ -1,8 +1,67 @@
 ---
 title: "Minimal KubeSphere on Kubernetes"
-keywords: 'kubesphere, kubernetes, docker, multi-tenant'
-description: 'Install a Minimal KubeSphere on Kubernetes'
+keywords: 'KubeSphere, Kubernetes, Minimal, Installation'
+description: 'Minimal Installation of KubeSphere on Kubernetes'
 
 linkTitle: "Minimal KubeSphere on Kubernetes"
 weight: 3020
 ---
+
+In addition to installing KubeSphere on a Linux machine, you can also deploy it on existing Kubernetes clusters directly. This QuickStart guide walks you through the general steps of completing a minimal KubeSphere installation on Kubernetes. For more information, see [Installing on Kubernetes](https://kubesphere-v3.netlify.app/docs/installing-on-kubernetes/).
+
+{{< notice note >}}
+
+- To install KubeSphere on Kubernetes, your Kubernetes version must be `1.15.x, 1.16.x, 1.17.x, or 1.18.x`;
+- Make sure your machine meets the minimal hardware requirement: CPU > 1 Core, Memory > 2 G;
+- A default Storage Class in your Kubernetes cluster needs to be configured before the installation;
+- The CSR signing feature is activated in kube-apiserver when it is started with the `--cluster-signing-cert-file` and `--cluster-signing-key-file` parameters. See [RKE installation issue](https://github.com/kubesphere/kubesphere/issues/1925#issuecomment-591698309).
+- For more information about the prerequisites of installing KubeSphere on Kubernetes, see [Prerequisites](https://kubesphere-v3.netlify.app/docs/installing-on-kubernetes/introduction/prerequisites/).
+
+{{</ notice >}} 
+
+## Deploy KubeSphere
+
+After you make sure your machine meets the prerequisites, you can follow the steps below to install KubeSphere.
+
+- Execute the following commands to start installation:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kubesphere/ks-installer/master/deploy/kubesphere-installer.yaml
+```
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kubesphere/ks-installer/master/deploy/cluster-configuration.yaml
+```
+
+{{< notice note >}} 
```

Comment:
It would be better to move this Tip before this command:

```bash
kubectl apply -f https://raw.githubusercontent.com/kubesphere/ks-installer/master/deploy/cluster-configuration.yaml
```